examples/mod-source-record-manager/mappingMetadataDto.sample

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You can use this tool to format json:
https://jsonformatter.curiousconcept.com/#